dask_kubernetes/utils.py

```diff
@@ -41,7 +41,7 @@ def namespace_default():
 
 
 async def get_external_address_for_scheduler_service(
-    core_api, service, port_forward_cluster_ip=None
+    core_api, service, port_forward_cluster_ip=None, service_name_resolution_retries = 20
```

Comment:
I think it makes sense and would like to expose `service_name_resolution_retries` as a config option when instantiating `KubeCluster`. 

If you agree, any suggestion on the best way to do this is appreciated @jacobtomlinson